Terminal-iOS/MyStudy/StudyDetail/ViewComponent/ApplyList/MyApplyListCell.swift

```diff
@@ -20,15 +20,44 @@ class MyApplyListCell: ApplyListCell {
             $0.selectionStyle = .none
         }
         self.mainImage.do {
-            $0.layer.cornerRadius = 20
+            $0.layer.cornerRadius = 10
             $0.layer.masksToBounds = true
             $0.clipsToBounds = true
             $0.contentMode = .scaleAspectFill
         }
     }
     func setData(studies: ApplyStudy) {
         if let imageURL = studies.image {
```

Comment:
```suggestion
        let imageURL = studies.image ?? "" 
```
요런식으로 하면 공백만 체크해도 되지 않을까 싶네요!!